picard/disc/utils.py

```diff
@@ -50,12 +50,12 @@ def calculate_mb_toc_numbers(toc):
     toc = _remove_data_track(toc)
     num_tracks = len(toc)
     if not num_tracks:
-        raise NotSupportedTOCError("Empty track list: %s", toc)
+        raise NotSupportedTOCError("Empty track list")
 
     expected_tracknums = tuple(range(1, num_tracks+1))
     tracknums = tuple(e.number for e in toc)
     if expected_tracknums != tracknums:
-        raise NotSupportedTOCError("Non-standard track number sequence: %s", tracknums)
+        raise NotSupportedTOCError(f"Non-standard track number sequence: {tracknums}")
```

Comment:
Oh yes, we have Python 3.6 declared as the minimum supported version and hence can use f strings. Good move